.github/workflows/release.yaml

```diff
@@ -4,17 +4,17 @@ on:
   workflow_dispatch:
     inputs:
       destination-channel:
-        description: 'Destination Channel'
+        description: "Destination Channel"
         required: true
       origin-channel:
-        description: 'Origin Channel'
+        description: "Origin Channel"
         required: false
       rev:
-        description: 'Revision number'
+        description: "Revision number"
```

Comment:
They are preferred to be mutually exclusive, but I don't know of a way to enforce it in the yaml. If both are provided, rev would be used

Reply:
Yeah in the workflow_dispatch yaml I dont know of a way to enforce that.  I think the only way to do it would be in the action itself. 

I propose updating `release-charm` to enforce the mutual exclusivity and error out if both are set.  They should never be set at the same time (unless I misunderstand - please correct if I do), so specifying both sounds like a mistake that should warrant an error.  Then in the UI here, you can add to the argument descriptions that exactly one of `Revision Number` and `Origin Channel must be set

Reply:
@DomFleischmann what's your thought on this?

Reply:
I'm marking this as resolved and suggest we continue the discussion [here](https://github.com/canonical/charming-actions/pull/38#discussion_r844086882) which feels more appropriate